packages/react/src/components/Authenticator/hooks/useAuthenticator/index.tsx

```diff
@@ -52,9 +52,15 @@ export const Provider = ({ children }) => {
     service: { send },
   } = value;
 
+  const isListening = React.useRef(false);
   React.useEffect(() => {
+    if (isListening.current) {
+      return;
+    }
+
+    isListening.current = true;
     return listenToAuthHub(send);
-  }, []);
+  }, [send]);
```

Comment:
Is this because of the exhaustive deps eslint rule? So it only runs once?

Reply:
That's correct.